WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -186,7 +214,7 @@ private extension ItemListView {
         )
         static let headerBannerHint = NSLocalizedString(
             "pos.itemlistview.headerBannerHint",
-            value: "Other product types, such as variable and virtual, will become available in future updates.",
+            value: "Other product types, such as variable and virtual, will become available in future updates. Learn more",
```

Comment:
Do we know what we want to open with this "Learn more"? I think we can skip this change until we implement that.

Reply:
I will check if that also opens the full banner or something else!